api/URL.json

```diff
@@ -87,9 +87,15 @@
               "prefix": "webkit"
             }
           ],
-          "samsunginternet_android": {
-            "version_added": true
-          },
+          "samsunginternet_android": [
```

Comment:
Where did the Samsung Internet data come from?

Reply:
Since Samsung Internet is a Chrome derivative, I copied the data from there!